backend/src/backend/main.py

```diff
@@ -97,25 +103,37 @@ async def add_book(book_payload: BookAddSchema):
                 kind=KIND_TEXT,
             )
 
-    title = await Title.objects.get_or_create(
-        ident=get_ident_from_name(book_payload.metadata["Name"])
-    )
-    await book.update(title=title)
+    for metadata_name, value in book_payload.metadata.items():
```

Comment:
Would make more sense to loop over the BookMetadata to create the BookMetadata. It thus becomes a simple copy and you don't have to do different stuff for illustration.